3_acm/root-repo.yaml.tpl

```diff
@@ -0,0 +1,14 @@
+apiVersion: configsync.gke.io/v1alpha1
+kind: RootSync
+metadata:
+  name: root-sync
+  namespace: config-management-system
+spec:
+  sourceFormat: hierarchy
+  git:
+    repo: git@${GITLAB_HOSTNAME}:platform-admins/anthos-config-management.git
+    branch: master
+    dir: clusters/${CONTEXT}
```

Comment:
We can remove `dir` as it will be the default root directory of the repo.

Reply:
Good catch! It is not needed.